provisioning/playbooks/couchbase/install_couchbase.yml

```diff
@@ -0,0 +1,18 @@
+---
+# This installs the Couchbase Server software only and does not
+# actually initialize a cluster, see cluster_init.yml for that
+
+- hosts: couchbase
+  sudo: True
+  roles:
+    - common
+    - couchbase
+  tasks:
+    - name: Couchbase Server Service
+      service: name=couchbase-server state=started enabled=yes
+
+    - name: Wait for Admin
+      wait_for: port=8091 delay=20
+
+    - name: Wait for EPMD
```

Comment:
Couchbase uses Erlang to communicate between nodes of a cluster. EPMD means Erlang Port Mapper Daemon. Could remove this bit because this is a single node Couchbase cluster.